unlock-app/src/components/interface/Header.js

```diff
@@ -40,8 +71,64 @@ const TopHeader = styled.header`
   grid-auto-flow: column;
   align-items: center;
   height: 70px;
+
+  @media (max-width: 600px) {
+    display: flex;
```

Comment:
Would there be a way to use `grid` here too to be consistent with the rest of the UI?